include/oboe/AudioStreamCallback.h

```diff
@@ -108,26 +109,29 @@ class AudioStreamErrorCallback {
      * Or it might be used when an app want to signal a management thread that handles
      * all of the stream state.
      *
-     * If this method returns false then
-     * the stream will be stopped by Oboe in the following way: onErrorBeforeClose() will be called,
-     * then the stream will be closed and onErrorAfterClose() will be closed.
+     * If this method returns false it indicates that the stream has *not been stopped and closed
+     * by the application. In this case it will be stopped by Oboe in the following way:
+     * onErrorBeforeClose() will be called, then the stream will be closed and onErrorAfterClose()
+     * will be closed.
      *
-     * If this method returns true then the normal error processing will not occur.
-     * In that case, the app MUST stop() and close() the stream!
+     * If this method returns true it indicates that the stream *has* been stopped and closed
+     * by the application and the normal error processing will *not* occur.
+     * In that case, the app MUST stop() and close() the stream.
      *
-     * Note that this method will be called on a thread created by Oboe.
+     * This method will be called on a thread created by Oboe.
      *
      * @param audioStream pointer to the associated stream
      * @param error
-     * @return true if the error has been handled, false if not
+     * @return true if the stream has been stopped and closed,
+     * false if not
```

Comment:
Will this get picked up by Doxygen?

Reply:
Moved back onto single line to be on the safe side.